WebLibs/api/fluentRestClient.js

```diff
@@ -132,7 +132,9 @@ export function restClient() {
                     return await res.json();
                 }
 
-                if (contentType.includes("image") || contentType.includes("application/pdf")) {
+                const blobTypes = ["pdf", "zip", "octet-stream"];
```

Comment:
@gdl-reif: Are we using XML files? Not sure if we should add `xml` here as well.

Reply:
We will need support for XML, both serving it from our APIs and consuming it, but right now it's not top prio